crm_lead_to_event_registration/wizard/crm_lead_event_pick.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class WizardModel(models.TransientModel):
+    _name = "crm.lead.event.pick"
+
+    lead_id = fields.Many2one(
+        "crm.lead",
+        "Lead/Opportunity",
+        default=lambda self: self.env.context["active_id"],
+        help="Lead/Opportunity from where to get contact information.")
+    event_id = fields.Many2one(
+        "event.event",
+        "Event",
+        required=True,
+        help="Event where the registration will be created.")
+
+    @api.multi
+    def action_accept(self):
+        self.lead_id.action_generate_event_registration(self.event_id)
```

Comment:
Should this method support multirecord? (because of api.multi):
```
@api.multi
def action_accept(self):
    for wizard in self:
        wizard.lead_id.action_generate_event_registration(wizard.event_id)